mina.netty/src/main/java/org/jboss/netty/channel/socket/nio/AbstractNioWorker.java

```diff
@@ -629,9 +626,13 @@ public void run() {
                     int interestOps = channel.getInternalInterestOps();
                     interestOps |= SelectionKey.OP_WRITE;
                     channel.setInternalInterestOps(interestOps);
-                    ReadDispatcher readDispatcher = channel instanceof NioSocketChannel
-                            ? new TcpReadDispatcher((NioSocketChannel) channel)
-                            : new UdpReadDispatcher((NioDatagramChannel) channel);
+                    ReadDispatcher readDispatcher = null;
```

Comment:
please format and include brackets on the ifs, this is especially ugly because you indent on the if to where the bracket would be as if the else if is suppose to be under the general if statement